src/oaklib/converters/owl_to_fhir.py

```diff
@@ -0,0 +1,20 @@
+"""Convert OWL to FHIR CodeSystem JSON
+
+TODO's
+  1. Write functioning converter
+  2. Schema mappings (Obograph -> FHIR CodeSystem). SchemaAutomator? Other approach?
+  3. Add converter to CLI
+  4. Test(s)
+
+todo's (minor)
+  1. owl_to_fhir.py -> owl_to_fhir_codesystem.py?
+"""
+
+
+def convert(inpath: str):
```

Comment:
Just going to do things quick and dirty at first, as opposed to trying to conform to codebase conventions straight off the bat.